lib/local-links-manager/check_links/homepage_status_updater.rb

```diff
@@ -21,7 +21,7 @@ def update
     private
 
       def links
-        table.distinct.pluck(column)
+        table.distinct.where.not(column => '').pluck(column)
```

Comment:
This *might* be database dependent as I think mysql might treat `''` and `null` differently.  I'd need to check it against a mysql db - but happily we're not using mysql so no need to worry.  Just pointing it out for future reference.  The good thing here is that we have the tests that explain what we actually want so if we did switch dbs we'd spot the failure and change the implementation.  Good stuff!